manifests/config.pp

```diff
@@ -53,7 +61,7 @@
     require => User[$caddy::caddy_user],
   }
 
-  case $::operatingsystemmajrelease {
+  case $operatingsystemmajrelease {
```

Comment:
this won't work, because it's a fact.Please undo the change.

Reply:
done